python/tutorials/10-block-scaled-matmul.py

```diff
@@ -73,14 +119,15 @@
 import triton.profiler as proton
 from triton.tools.tensor_descriptor import TensorDescriptor
 from triton.tools.mxfp import MXFP4Tensor, MXScaleTensor
+from triton._internal_testing import is_hip_cdna4
```

Comment:
Let's not import from `_internal_testing` for tutorials, which means to be hermetic on its own for demo purposes. Redefine like what's there for cuda.